extensions/wikia/PhalanxII/fallback/hooks/ContentBlock.class.php

```diff
@@ -15,42 +15,42 @@ public static function onEditFilter( $textbox, $summary, &$block ) {
 		global $wgOut, $wgTitle;
 		wfProfileIn( __METHOD__ );
 
-		//allow blocked words to be added to whitelist
-		if ($wgTitle->getPrefixedText() == 'MediaWiki:Spam-whitelist') {
+		// allow blocked words to be added to whitelist
+		if ( $wgTitle->getPrefixedText() == 'MediaWiki:Spam-whitelist' ) {
 			wfProfileOut( __METHOD__ );
 			return true;
 		}
 
 		// here we get only the phrases for blocking in summaries...
 		$blocksData = PhalanxFallback::getFromFilter( PhalanxFallback::TYPE_SUMMARY );
-		if ( !empty($blocksData) && $summary != '' ) {
-			$summary = self::applyWhitelist($summary);
+		if ( !empty( $blocksData ) && $summary != '' ) {
+			$summary = self::applyWhitelist( $summary );
 
 			$blockData = null;
-			$result = PhalanxFallback::findBlocked($summary, $blocksData, true, $blockData);
+			$result = PhalanxFallback::findBlocked( $summary, $blocksData, true, $blockData );
 			if ( $result['blocked'] ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "blocked" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)